### DISCUSSION TITLE
More than 1 note page per day possible?

### DISCUSSION THREAD

**Discussion by Onkeyuk** [kudrykv/latex-yearly-planner#80] [General]
First, thank you for this, it’s excellent.

I’m using the daily planner but I take a lot of notes per day. Is it possible to add more notes pages per day? Say 4?

**Comment by kudrykv** [comment]
[sn_a5x.mos.dotted.default.dailycal.2023.pdf](https://github.com/kudrykv/latex-yearly-planner/files/11301108/sn_a5x.mos.dotted.default.dailycal.2023.pdf)
[sn_a5x.mos.dotted.default.dailycal.2024.pdf](https://github.com/kudrykv/latex-yearly-planner/files/11301109/sn_a5x.mos.dotted.default.dailycal.2024.pdf)

**Reply by Onkeyuk** [reply]
Thanks so much. Would this work in a remarkable 2?

**Reply by kudrykv** [reply]
The difference between "devices" are some modifications to the layout.

The last time I checked, Remarkable was having menu bar on the left. Does this still hold true? Can you move the bar to the other sides?

Anyway, here are rm versions
[rm2.mos.dotted.default.dailycal.2023.pdf](https://github.com/kudrykv/latex-yearly-planner/files/11301412/rm2.mos.dotted.default.dailycal.2023.pdf)
[rm2.mos.dotted.default.dailycal.2024.pdf](https://github.com/kudrykv/latex-yearly-planner/files/11301413/rm2.mos.dotted.default.dailycal.2024.pdf)

**Reply by Onkeyuk** [reply]
Really appreciate that. Yes the remarkable has the menu on the left that can be hidden but not moved. 
I would love a Supernote but unfortunately it is twice the price to get delivered here

**Reply by If-You-Will** [reply]
@kudrykv

I only recently found this daily planner and can't say enough good things about it.

I have been able to setup an environment where I can successfully create a planner following the instructions you have provided.  I am currently experimenting and attempting to tweak a few things, such as what you provided here with the extra pages.  

If it isn't too much trouble, are you able to share the cfg or tpls used in creating the remarkable planner with the extra pages (rm2.mos.dotted.default.dailycal.2023.pdf).  This will provide me with some direction in how to customize this further.

Thanks again for all your help.

**Reply by kudrykv** [reply]
Hi @If-You-Will, thank you for kind words!

You'll need to throw a loop in here to have more pages. It's not driven through the configuration (at least, not yet) https://github.com/kudrykv/latex-yearly-planner/blob/a321d94fc4edf271ac564b232d356978b91578ed/app/compose/daily.go#L22-L41

**Reply by nekoskyex** [reply]
hi there, i have been searching for the perfect planner and this seems to be promising! i am using the file [sn_a5x.mos.dotted.default.dailycal.2023.pdf](https://github.com/kudrykv/latex-yearly-planner/files/11301108/sn_a5x.mos.dotted.default.dailycal.2023.pdf) and i am struggling to figure out how to replicate this with lines instead of dots. i am very new to code and such but have been able to install everything, and adjust the number of note pages, etc.

what i am stuck on is the header style with the backward and forward arrows around the date on the left side instead of the date breadcrumbs off to the right, and with the week, calendar, and notes links there instead. also the side tabs like this file has as well. 

i have been successful in converting the dots to lines, adjusting the schedule times to work for me, and reducing the notes pages from 4 to 2 per day instead. 

any help or guidance on the breadcrumbs and side tabs would be very much appreciated!

**Reply by nekoskyex** [reply]
i figured out the first part, but now i am stuck on the margins. i believe I need to be adjusting this part of the base.yaml file. i have played around with different numbers here for the marginparwidth and left margins but this is probably a rookie misunderstanding. 

```
layout:
  paper:
    width: 15.6cm
    height: 20.9cm

    marginparwidth: 1cm
    marginparsep: 0cm
    reversemargins: true

    margin:
      top: 1.2cm
      bottom: 0.7cm
      left: 0.5cm
      right: 0.5cm
```

here is the closest i can seem to get without lines intersecting into the tabs.

<img width="213" alt="CleanShot 2023-08-10 at 11 22 08@2x" src="https://github.com/kudrykv/latex-yearly-planner/assets/67119172/dc775d93-5204-41b5-8831-8b374886969a">

any assistance is very much appreciated.

**Reply by kudrykv** [reply]
@nekoskyler hi! 

> figure out how to replicate this with lines instead of dots

There is one with lines instead of dots. Do you miss anything in it?
https://github.com/kudrykv/latex-yearly-planner/releases/download/v0.0.8.2324/sn_a5x.mos.lined.default.dailycal.2023.pdf

**Reply by nekoskyex** [reply]
@kudrykv Thanks for the reply! I didn't see this one, actually, but I am trying to replicate it on my end so I can produce them year after year on my own. I did figure out how to get the lines on my own, and I also needed to adjust the times and wanted the extra note pages, which I was able to figure out as well. The only thing I don't understand is how to adjust the margins in the code so that the schedule isn't right up against the date tabs. The date tabs also look smaller when I run it without any edits, as if part of the tab is cut off; I was hoping it was an easy thing to fix.

**Reply by kudrykv** [reply]
@nekoskyler ah, gotcha.

I would suggest, though, starting with a predefined lined config and editing from there — it _should_ be easier.

> how to adjust the margins in the code so that the schedule isn't right up against the date tabs

It's somewhere on the Go/LaTeX edge. Or, maybe, even closer to LaTeX. Unfortunately, Latex is never easy.

You should check `marginparwidth` / `marginparsep` variables.
https://github.com/kudrykv/latex-yearly-planner/blob/a321d94fc4edf271ac564b232d356978b91578ed/tpls/document.tpl#L38-L39

![CleanShot 2023-08-15 at 11 53 03](https://github.com/kudrykv/latex-yearly-planner/assets/12697803/442e1f28-5402-403a-b83c-391d7952cb13)
https://www.overleaf.com/learn/latex/Articles/A_visual_guide_to_LaTeX%E2%80%99s_page_layout_parameters

**Reply by harisez** [reply]
Is it possible to get a line version instead of the dotted version ?

Feel free to point me to steps to do this too.

Thanks!

**Reply by kudrykv** [reply]
`dotted: false` should do
https://github.com/kudrykv/latex-yearly-planner/blob/main/cfg/base.yaml#L3

**Comment by If-You-Will** [comment]
@kudrykv

Thank you so much for pointing me in the right direction.  I modified the func slightly to include a page count and then added the loop as you suggested.  This appears to be working.

```
var Daily = DailyStuff("", "",1)
var DailyReflect = DailyStuff("Reflect", "Reflect",1)
var DailyNotes = DailyStuff("More", "Notes",2)

func DailyStuff(prefix, leaf string, pages int) func(cfg config.Config, tpls []string) (page.Modules, error) {


//..... 

for i:=0; i < pages; i++{
					

if day.Time.IsZero() {
	continue
}

modules = append(modules, page.Module{
	Cfg: cfg,
	Tpl: tpls[0],
	Body: map[string]interface{}{
		"Year":         year,
		"Quarter":      quarter,
		"Month":        month,
		"Week":         week,
		"Day":          day,
		"Breadcrumb":   day.Breadcrumb(prefix, leaf, cfg.ClearTopRightCorner && len(leaf) > 0),
		"HeadingMOS":   day.HeadingMOS(prefix, leaf),
		"SideQuarters": year.SideQuarters(day.Quarter()),
		"SideMonths":   year.SideMonths(day.Month()),
		"Extra":        day.PrevNext(prefix).WithTopRightCorner(cfg.ClearTopRightCorner),
		"Extra2":       extra2(cfg.ClearTopRightCorner, false, false, week, 0),
	},
})
}
```

**Reply by fraggmented** [reply]
@If-You-Will - Could you assist? I'm having difficulties incorporating your code to produce ~4x note pages per day. I've tried replacing/adding the lines into the daily.go file, but it keeps producing an error.

**Reply by If-You-Will** [reply]
@fraggmented 

I will try and assist.  

var DailyNotes = DailyStuff("More", "Notes",2)

This line is calling the function DailyStuff .  The DailyStuff function has 3 parameters and the third (last parameter) is for the number of pages.  By changing the number 2 to whatever number you want should produce the extra pages.

**Reply by fraggmented** [reply]
@If-You-Will - I figured out your code, and worked out the error I was facing (missed a '}').

🙏 Thanks for the quick-assist!

**Comment by dcg1080** [comment]
I'm looking to replicate this and build on it for future planners. I'm learning (if you can call it that) as I go, so with limited knowledge I'm trying to discern first what is possible and how to do it if so. 

I see the templates of multiple note pages above, but they leave something to be desired in regard to navigation and linking. 

Is it possible that along with creating additional note pages per day, there could also be links on the Daily pages (so instead of "Notes|More" it would be something like "Notes|1 2 3 4 5")? I've learned enough to format the page that way, but not how to create the links. 

Additionally, navigation on the notes pages is difficult without additional links since the standard header is "[Yesterday] | [Tomorrow]" which skips the additional note pages. So is it possible to add additional navigation links on those pages along the lines of 1,2,3,... at the top to navigate additional note pages on the same date, or possible something simpler like "[Yesterday] | < > | [Tomorrow]" where the arrows would scroll notes for that date one at a time and the before and after dates would jump to another?

(I'm tinkering, but odds of my figuring it out I recognize as very slim.)

Really love this planner. Is the old coffee links still active?

**Reply by kudrykv** [reply]
Check out `rubify2` branch, it's easier to tinker with. Only MOS template is implemented there, though.

> Is it possible that along with creating additional note pages per day, there could also be links on the Daily pages (so instead of "Notes|More" it would be something like "Notes|1 2 3 4 5")? I've learned enough to format the page that way, but not how to create the links.

Config number of days: https://github.com/kudrykv/latex-yearly-planner/blob/50e302e225840051f4340033499110832d063c95/config/119x158-rightie.yaml#L126-L129
Use "More 2 3 4 5" for output: https://github.com/kudrykv/latex-yearly-planner/blob/50e302e225840051f4340033499110832d063c95/config/119x158-rightie.yaml#L124
Place where "More 2 3 4 5" builds: https://github.com/kudrykv/latex-yearly-planner/blob/50e302e225840051f4340033499110832d063c95/lib/latex_yearly_planner/planners/mos/pages/daily_callable.rb#L92-L98

> Additionally, navigation on the notes pages is difficult without additional links since the standard header is "[Yesterday] | [Tomorrow]" which skips the additional note pages

Makes sense. It's possible, although it'll require some thinking. Won't be super-fast to tackle.

Yes, coffee links are still active.

**Reply by dcg1080** [reply]
Thanks! I played with the main branch for a while before realizing I probably should have started in the rubify branch.
But by that point I had figured the majority of what I wanted to do and created a 2025 planner.

Got my held over for now and I'll find more time to play with these options (and learn a lot as I go).

Can't say thanks enough for all the work you've done.
